pyomo/contrib/fme/tests/test_fourier_motzkin_elimination.py

```diff
@@ -20,8 +20,14 @@
 from pyomo.gdp import Disjunction, Disjunct
 from pyomo.repn.standard_repn import generate_standard_repn
 from pyomo.core.kernel.component_set import ComponentSet
+import random
 
 class TestFourierMotzkinElimination(unittest.TestCase):
```

Comment:
Is this here intentionally, Emma? I think FME is in your other PR?

Reply:
Yes, but only because I broke the FME tests with changes to the transformation block.